src/cart/crt_hg_proc.c

```diff
@@ -712,14 +712,9 @@ crt_proc_out_common(crt_proc_t proc, crt_rpc_output_t *data)
 
 		rc2 = rpc_priv->crp_reply_hdr.cch_rc;
 		if (rc2 != 0) {
-			if (rpc_priv->crp_reply_hdr.cch_rc != -DER_GRPVER)
-				RPC_ERROR(rpc_priv,
-					  "RPC failed to execute on target. "
-					  "error code: "DF_RC"\n", DP_RC(rc2));
-			else
-				RPC_TRACE(DB_NET, rpc_priv,
-					  "RPC failed to execute on target. "
-					  "error code: "DF_RC"\n", DP_RC(rc2));
+			RPC_WARN(rpc_priv,
+				 "RPC failed to execute on target. "
+				 "error code: "DF_RC"\n", DP_RC(rc2));
```

Comment:
same as above, you dont really want to print these as warns for DER_GRPVER either